packages/massa-proto-cli/dist/as-gen.d.ts

```diff
@@ -0,0 +1,2 @@
+import { IProtoFile } from './protobuf';
```

Comment:
Should be ProtoFile, isnt' it ?
```suggestion
import { ProtoFile } from './protobuf';
```